src/main/java/ch/saunah/saunahbackend/service/GoogleCalendarService.java

```diff
@@ -156,4 +186,16 @@ public void updateEvent  (String calenderID ,String eventID , DateTime startDate
 
         service.events().update(calenderID, event.getId(), event).execute();
     }
+
+    /**
+     * Confirm a event on the google calender
+     * @param calenderID Id of the google Calendar
+     * @param eventID id of the event
+     * @throws IOException
+     */
+    public void approveEvent(String calenderID ,String eventID) throws IOException{
+        Event event =service.events().get(calenderID, eventID).execute();
+        event.setStatus(CONFIRMED);
```

Comment:
Is this displayed anywhere on the calendar? I could not find it when I tested, so not sure if it's needed :)

Reply:
No it's not displayed , but it isnt nesscary anymore if we change the event name to "Provisiorisch" and "Gebucht" as so mentioned before